src/components/pages/index/Biography/index.js

```diff
@@ -10,9 +10,9 @@ const Biography = () => (
       <div className='row middle-xs'>
         <div className='col-xs-12 col-sm-6'>
           <h3 className={styles.biographyText}>
-            I'm a 24 years old <b>Front End Engineer</b> based in <b>Barcelona</b>,
+            I'm a 24 year old <b>Front End Engineer</b> based in <b>Barcelona</b>,
             working at <a href='https://ulabox.com' rel='noopener noreferrer'><b>Ulabox</b></a>.
-            I'm a coding passionate that is focused on <b>JavaScript</b>, specifically in React & React-Native.
+            I'm a passionate coder focused on <b>JavaScript</b>, specifically React & React-Native.
```

Comment:
Why this is not correct ?

```suggestion
            I'm a coding passionate focused on <b>JavaScript</b>, specifically React & React-Native.
```

Reply:
Good question!

We don't have `passionate` as a noun in English only as an adjective.

You could say your a `coding enthusiast` if you wanted to.